components/script/dom/dissimilaroriginwindow.rs

```diff
@@ -158,7 +158,8 @@ impl DissimilarOriginWindowMethods for DissimilarOriginWindow {
 
         // Step 1-2, 6-8.
         // TODO(#12717): Should implement the `transfer` argument.
-        let data = StructuredCloneData::write(cx, message)?;
+        rooted!(in(cx) let transfer = UndefinedValue());
+        let data = StructuredCloneData::write(cx, message, transfer.handle())?;
```

Comment:
could the TODO on line 160 now be removed?

Reply:
I don't think so, because the transfer argument is undefined. The proper way to do this would be to actually obtain them from the arguments passed to `window.postMessage`.

Reply:
Ok, I see